docs/prestosql-guide.md

```diff
@@ -139,11 +139,15 @@ Then, `query.max-memory=<30GB * number of nodes>`.
 
 ##### Configuring Carbondata in Presto
 1. Create a file named `carbondata.properties` in the `catalog` folder and set the required properties on all the nodes.
+2. As carbondata connector extends, hive connector. All the configurations(including S3) is same as hive connector.
+Just replace the connector name in hive configuration and make a copy of it to carbondata.properties
+`connector.name = carbondata`
```

Comment:
"make a copy of it to carbondata.properties" can be changed to "copy same to carbondata.properties"

Reply:
done